editor/editor_log.h

```diff
@@ -48,6 +48,7 @@ class EditorLog : public HBoxContainer {
 	enum MessageType {
 		MSG_TYPE_STD,
 		MSG_TYPE_ERROR,
+		MSG_TYPE_RICH,
 		MSG_TYPE_WARNING,
 		MSG_TYPE_EDITOR,
 	};
```

Comment:
Same as above: isn't `MSG_TYPE_RICH` just a "subtype" of `MSG_TYPE_STD`? If that's the case then maybe naming it `MSG_TYPE_STD_RICH` would make more sense?

Reply:
Made the changes and pushed out the rebase. Now hoping that the workflows will still all be positive hahaha, seeing those run is starting to give me stress hahaha

Reply:
Based on [your other comment](https://github.com/godotengine/godot/pull/60675#issuecomment-1114522170) it seems like you're trigerring format checks manually. You can [add pre-commit hooks](https://docs.godotengine.org/en/3.4/community/contributing/code_style_guidelines.html#pre-commit-hook) so such checks will be automatically made locally  for you whenever you commit. This way you won't be afraid of failing static checks (not sure what else could fail after such simple changes :wink:).

Reply:
@kleonc I didn't know about those pre-commit hooks, I always just used the clang_format.sh, I will take this in mind next time. I have been learning a lot about git and github recently. 
I've got one more question, now the workflow checks are completed, how does the process go from here? Just wait until someone approves the merge or gives comments if things needs to change?

Reply:
>I've got one more question, now the workflow checks are completed, how does the process go from here? Just wait until someone approves the merge or gives comments if things needs to change?

You could just wait or you could ask for a review on the [Godot Contributors Chat](https://chat.godotengine.org) (probably in the `general` channel). If some core contributor(s) would approve and no one would have anything against then who knows maybe it could be merged even today.